src/routes/Router/Router.tsx

```diff
@@ -1,43 +1,56 @@
-import { createBrowserRouter } from 'react-router-dom';
+import { createBrowserRouter, createRoutesFromElements, Route } from 'react-router-dom';
+import * as React from 'react';
 
 import { ErrorPage } from '@/resources/ErrorPage/ErrorPage';
-import { NewApiDelegationsPage } from '@/components/NewApiDelegationPage';
+import { NewApiDelegationPage } from '@/components/NewApiDelegationPage';
 import { OverviewPage as GivenOverviewPage } from '@/components/apiDelegation/given/OverviewPage';
 import { OverviewPage as ReceivedOverviewPage } from '@/components/apiDelegation/received/OverviewPage';
 import { NewOrgDelegationPage } from '@/components/NewOrgDelegationPage';
 import { ApiDelegationConfirmationPage } from '@/components/ApiDelegationConfirmationPage';
 
+export enum RouterPath {
+  GivenApiDelegations = 'given-api-delegations',
+  GivenApiDelegationOverview = 'overview',
+  NewGivenApiDelegation = 'pick-api',
+  NewGivenOrgDelegation = 'pick-org',
+  GivenApiDelegationConfirmation = 'confirmation',
+  GivenApiDelegationReceipt = 'receipt',
+  ReceivedApiDelegations = 'received-api-delegations',
+  ReceivedApiDelegationOverview = 'overview',
+  Profile = 'Profile',
+}
+
 export const Router = createBrowserRouter(
-  [
-    {
-      path: '/',
-      errorElement: <ErrorPage />,
-    },
-    {
-      path: 'api-delegations',
-      element: <GivenOverviewPage />,
-      errorElement: <ErrorPage />,
-    },
-    {
-      path: 'received-api-delegations',
-      element: <ReceivedOverviewPage />,
-      errorElement: <ErrorPage />,
-    },
-    {
-      path: 'api-delegations/new-api-delegation',
-      element: <NewApiDelegationsPage />,
-      errorElement: <ErrorPage />,
-    },
-    {
-      path: 'api-delegations/new-org-delegation',
-      element: <NewOrgDelegationPage />,
-      errorElement: <ErrorPage />,
-    },
-    {
-      path: 'api-delegations/confirmation',
-      element: <ApiDelegationConfirmationPage />,
-      errorElement: <ErrorPage />,
-    },
-  ],
+  createRoutesFromElements(
+    <Route
+      path='/'
+      errorElement={<ErrorPage />}
+    >
+      <Route
+        path={RouterPath.GivenApiDelegations + '/' + RouterPath.GivenApiDelegationOverview}
+        element={<GivenOverviewPage />}
+      />
```

Comment:
Why does this page not have a `errorElement={<ErrorPage />}` when all the others do?

Reply:
Because I forgot it :p